functions/_fzf_search_git_log.fish

```diff
@@ -4,9 +4,9 @@ function _fzf_search_git_log --description "Search the output of git log and pre
     else
         # see documentation for git format placeholders at https://git-scm.com/docs/git-log#Documentation/git-log.txt-emnem
         # %h gives you the abbreviated commit hash, which is useful for saving screen space, but we will have to expand it later below
-        set log_fmt_str '%C(bold blue)%h%C(reset) - %C(cyan)%ad%C(reset) %C(yellow)%d%C(reset) %C(normal)%s%C(reset)  %C(dim normal)[%an]%C(reset)'
+        set log_fmt_str '%C(bold blue)%h%C(reset) - [%G?] - %C(cyan)%ad%C(reset) %C(yellow)%d%C(reset) %C(normal)%s%C(reset)  %C(dim normal)[%an]%C(reset)'
```

Comment:
Let's leave this part out. I suspect most people won't care whether it was verified or not. I will add to my todo list making the log format string customizable so you can have this.

Reply:
And we can't add like a parameter to configure if we want it to appear or not and default it to not appear? I agree most people won't care but I would like to have the option to keep it

Reply:
> I will add to my todo list making the log format string customizable so you can have this.

Reply:
Wow it was just two sentences and I missed one of them. Sorry, I'll take the flag out

Reply:
As promised! 039a86de01b43500bcc7af7b3a188e9093098cd1